src/ophyd_async/epics/adcore/_utils.py

```diff
@@ -7,11 +7,12 @@
     SignalRW,
     StrictEnum,
     SubsetEnum,
+    SupersetEnum,
     wait_for_value,
 )
 
 
-class ADBaseDataType(StrictEnum):
+class ADBaseDataType(SupersetEnum):
```

Comment:
Should probably allow the empty case here as we tend to blank entries the driver doesn't support...
```suggestion
class ADBaseDataType(SupersetEnum):
    _ = ""  # Driver database override will blank the enum string if it doesn't support a datatype
```
We then need to handle this in the functions below